packages/accordion/src/types.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import type { HTMLProps } from 'react';
+
+export interface IUseAccordionProps<Value> {
+  /** Prefixes IDs for the accordion trigger and panels  */
+  idPrefix?: string;
+  /** Defines the sections for the accordion */
+  sections: Value[];
+  /** Sets the expanded sections in a controlled accordion */
+  expandedSections?: Value[];
+  /** Sets the default expanded sections in a uncontrolled accordion */
+  defaultExpandedSections?: Value[];
+  /** Handles accordion expansion changes */
+  onChange?: (expanded: Value) => any;
+  /** Determines if multiple panels can be expanded at the same time in an uncontrolled accordion */
+  expandable?: boolean;
+  /** Determines if panels can be collapsed in an uncontrolled accordion */
+  collapsible?: boolean;
+}
+
+export interface IUseAccordionReturnValue<Value> {
+  expandedSections: Value[];
+  disabledSections: Value[];
+  getHeaderProps: <T extends Element>(
+    props?: Omit<HTMLProps<T>, 'role' | 'aria-level'> & {
+      role?: 'heading' | null;
+      'aria-level': string | number | null;
+    }
+  ) => HTMLProps<T>;
+  getTriggerProps: <T extends Element>(
+    props?: Omit<HTMLProps<T>, 'role' | 'tabIndex'> & {
+      role?: 'button' | null;
+      tabIndex?: any | null;
+      value?: Value;
+    }
```

Comment:
```suggestion
    props: Omit<HTMLProps<T>, 'role'> & {
      role?: 'button' | null;
      value: Value;
    }
```

Seems like we're carrying forward a TS mistake from the original code. This `tabIndex` line should be deleted. Also, we need to make sure `value` is provided.